templates/zerver/help/send-group-pm.md

```diff
@@ -0,0 +1,72 @@
+# Group Private Messages
+Users can send group private messages in Zulip.
+## There are three ways to send a Group Private Message:
+### One:
```

Comment:
You can rephrase like this: There are three ways to send a private message to a group of people